Sources/AccessibilitySnapshot/Core/Swift/Classes/AccessibilitySnapshotView.swift

```diff
@@ -786,161 +785,6 @@ private extension AccessibilitySnapshotView {
 
 // MARK: -
 
-private extension UIView {
```

Comment:
Note for other reviewers: these methods are just being moved (without any edits) to new files